src/styles/pages/ussf-documentation.module.scss

```diff
@@ -1,40 +1,50 @@
 @import 'styles/uswdsDependencies';
 @import 'styles/sfds/sfdsDependencies';
 
-.accordion {
-  > h3 > button {
-    background-color: var(--accordion-btn-bg);
-    color: var(--accordion-title);
+.accordionGrid {
+  :global(.usa-accordion--bordered .usa-accordion__heading) {
+    margin-top: 0.5rem;
```

Comment:
This looks like a lot of changes, but I just needed to wrap everything in a new parent class, `accordionGrid`